content/en/blog/plugins-early-access.md

```diff
@@ -0,0 +1,157 @@
+
+The feature itself is still in very early access, and we expect it will be officially released early next year. In the meantime, we'd like to get potential plugin developers to take a look at the APIs and provide feedback while we move forward towards an official release.
+
+### What Are Plugins?
+
+Plugins are shared libraries that conform to a documented API and come in two flavors:
+
+* *Source Plugins* add new event sources/filter fields that can be evaluated using filtering expressions/Falco rules.
+* *Extractor Plugins* add the ability to define new fields that can extract information from either core Falco events or events generated by other plugins.
+
+Together, plugins should dramatically extend the functionality of Falco, allowing users to write rules for almost any kind of streaming data.
+
+A plugin can be written in any language, as long as the language supports exposing C function interfaces that can be called from a C/C++ program like Falco. We've written plugins in C, C++, and Go using this interface. For Go, we've also written a [module](https://pkg.go.dev/github.com/falcosecurity/plugin-sdk-go) that handles some of the work of translation between C pseudo-types and Go types, as well as providing convenience methods that make it easier to write plugins in Go.
+
+#### Source Plugins
+
+A source plugin provides a new event source. It has the ability to "open" and "close" a session that provides events. It also has the ability to return an event to the plugin framework via a `next()` method.
+
+Source plugins also have the ability to extract information from events based on fields. For example, a field (e.g. `proc.name`) extracts a value (e.g. process name like nginx) from a syscall event. The plugin returns a set of supported fields, and there are functions to extract a value given an event and field. The plugin framework can then build filtering expressions/Falco rule conditions based on these fields combined with relational and/or logical operators. For example, given an expression `ct.name=root and ct.region=us-east-1`, the plugin framework handles parsing the expression, calling the plugin to extract values for fields `ct.name`/`ct.region` for a given event, and determining the result of the expression. In a Falco output string like `An EC2 Node was created (name=%ct.name region=%ct.region)`, the plugin framework handles parsing the output string, calling the plugin to extract values for fields, and building the resolved string, replacing the template field names (e.g. `%ct.region`) with values (e.g. `us-east-1`).
```

Comment:
The "A field can extract a value from a syscall event" sentence ends up being misleading, because it implies that the only kind of event is a syscall event. I changed it to "For example, given a syscall event, a field `proc.name` can extract a process name such as `ngnix`". But otherwise, great suggestion and updated.